_pages/dev/seo/html-tags.md

```diff
@@ -0,0 +1,154 @@
+---
+title: HTML Tags
+feature:
+- name: HTML Tags
+  spa_version: 1.0
+  cx_version: n/a
+---
+
+HTML tags, and meta tags in particular, are used by search engines, social platforms, and bots to display page meta data on their platforms. By carefully preparing meta tags, and evaluating their values on a regular basis, you can improve the page ranking, click-through-rate, and usability of a page. All of this can affect SEO and the user experience.
```

Comment:
How about "to access page meta data in their platforms" ?

Because otherwise it becomes "... are _used_ by search engines [etc].... to _use_ page meta data..." , so we'd be using the verb "use" twice...

Reply:
I realized that "access" changes the meaning in a way we don't want. I've come up with this instead:

> HTML tags, and meta tags in particular, allow search engines, social platforms, and bots to use page meta data on their platforms.